src/cave-components/Cart/cart.jsx

```diff
@@ -0,0 +1,73 @@
+import "./cart.css";
+import "../../public-css/navbar.css";
+import "../../public-css/root.css";
+import { Link } from "react-router-dom";
+import { useCart } from "../../Context/cartContext"
```

Comment:
try using absolute path, instead of relative.
https://stackoverflow.com/questions/41993461/import-react-components-with-absolute-path

Reply:
Okay